product-features/liquidity-providers/faqs.md

```diff
@@ -0,0 +1,79 @@
+
+On the other hand, the **fee rebate** is the portion of the swap fee that is distributed to liquidity providers as a reward for supplying liquidity to the pool. The remaining portion of the swap fee goes to the ALEX platform. You can also find the fee rebate percentage in the Pool Info panel.
+
+![](../../.gitbook/assets/liquidity-providers/faqs-pool-info-panel-fees.png)
+
+</details>
+
+<details>
+
+<summary>What is the swap fee percentage that goes to liquidity providers?</summary>
+
+The swap fee percentage that goes to liquidity providers is known as the **fee rebate**. It is typically set at 50%, though it can vary depending on the pool. You can check this percentage in the Pool Info panel.
+
+</details>
+
+<details>
+
+<summary>What are the LP Tokens?</summary>
+
+Also know as "Pool Tokens", these tokens are issued to liquidity providers to represent their share of the liquidity pool. The total supply of LP tokens represents the 100% of the pool's funds.
+
+When users add liquidity to a pool, they receive LP tokens as proof of ownership. These tokens entitle them to a proportional share of the pooled assets and a portion of the fees generated by trades (swaps) within the pool.
+
+When liquidity is removed, the user transfers LP tokens back to the protocol. This determines how much of the pool's assets are returned to the user, along with their share of the transaction fees accrued during the time their liquidity was provided.
+
+</details>
+
+<details>
+
+<summary>How does the fee rebate benefit liquidity providers?</summary>
+
+The fee rebate is automatically reinvested into the pool, increasing the overall value of the pool. Since liquidity providers (LPs) hold a share of the pool, their holdings grow in value over time. However, these rewards can only be claimed when LP tokens (representing their share of liquidity) are withdrawn from the pool.
+
+</details>
+
+<details>
+
+<summary>How are fees transferred to the liquidity provider's wallet?</summary>
+
+The fees are not directly transferred to the liquidity provider's (LP) wallet. Instead, the swap fees allocated to LPs are reinvested into the liquidity pool. By holding LP tokens, liquidity providers accumulate their share of the fees over time. These accrued fees become available when they withdraw funds from the pool (i.e., when they remove liquidity). At that point, LP tokens are transferred back to the protocol, and in return, the provider receives their corresponding share of the pool's funds, including the accumulated fees.
+
+</details>
+
+<details>
+
+<summary>Where can I see the fees I've gained so far as a liquidity provider?</summary>
+
+You can view your accrued fees in the "My Liquidity" panel. To access it, navigate to the Swap -> Pool tab and select your pool of interest from the list. You'll also see a summarized version above the pool list.
```

Comment:
You can view your LP tokens in the "My Liquidity" panel. To access it, navigate to the Swap -> Pool tab and select your pool of interest from the list. You'll also see a summarized version above the pool list.

To Sofia:
You can not view your accrued fees in the " My Liquidity" Panel

Reply:
Yes, you're right. What do you think about this rephrasing? The complete answer for the question would be:

While there isn't a direct way to view your fees separately, you can check the "My Liquidity" panel for this purpose, which shows your LP tokens and liquidity provision details. To access it, navigate to the Swap -> Pool tab and select your pool of interest from the list. You'll also see a summarized version above the pool list.

In this panel, the **Pooled** amount reflects your total token holdings in the liquidity pool, which includes both your initial deposit and any fees you've accrued. Over time, this amount increases as more fees are added. The **Indicative Value** shows the USD equivalent of your holdings, which may fluctuate due to price changes of the pool's assets, but still provides a useful reference for tracking your gains.

Reply:
Done. Please resolve conversation or comment if any other change requested.